pipelines/defaults.json

```diff
@@ -2,6 +2,7 @@
     "repository"             : {
         "build_url"          : "https://github.com/AdoptOpenJDK/openjdk-build.git",
         "build_branch"       : "master",
+        "test_dirs"          : "/test/functional",
```

Comment:
Will need to update the `UsingOurScripts.md` documentation to explain this addition